bokehjs/src/lib/models/tools/math_text.ts

```diff
@@ -0,0 +1,130 @@
+import * as p from "core/properties"
+import {isNumber} from "core/util/types"
+import {Context2d} from "core/util/canvas"
+import {ImageLoader} from "core/util/image"
+import {CanvasImage} from "models/glyphs/image_url"
+import {Model} from "../../model"
+
+
+declare namespace MathJax {
+  function tex2svg(input: string): HTMLElement
+}
+
+export namespace MathText {
+  export type Attrs = p.AttrsOf<Props>
+
+  export type Props = Model.Props & {
+    text: p.Property<string>
+  }
+}
+
+export interface MathText extends MathText.Attrs {}
+
+type Position = {
+  sx: number
+  sy: number
+  x_anchor?: number | "left" | "center" | "right"
+  y_anchor?: number | "top"  | "center" | "baseline" | "bottom"
+}
+
+export class MathText extends Model {
+  override properties: MathText.Props
+
+  image: CanvasImage
+  height: number
+  width: number
+  mathjax_instantiated = false
```

Comment:
Members should be `protected` (or perhaps even `private`) if they are not intended to be a part of the public API of this model view hybrid.